1225-my_institution/361-institution_administrator_edits_my_institution.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Institution Administrator edits My Institution
+
+    @361
+    Scenario: Institution Administrator edits My Institution
+        Given that the user is logged in as Institution Administrator
+        And they click the menu item My Institution
+        When they edit fields:
+            | Name in organization registry |
+            | Display name                  |
+            | Short display name            |
+            | Archive name                  |
+            | Feide Organization ID         |

```

Comment:
```suggestion
```
ref https://github.com/BIBSYSDEV/NVA-scenarios/blob/master/1225-my_institution/360-institution_administrator_opens_my_institution.feature

Reply:
apropos: det blir litt smør på flesk om vi skal lage to scenario for alle forms, en for å liste opp felter og en egen for å lagre, men vi vil/må kanskje ha det slik..?

Reply:
går kanskje under det samme som vi fant ut for validering av forms, at vi burde hatt en `Feature` per form, som igjenhar flere `Scenario`, feks list opp felter, lagring, validering, etc

Reply:
er det kun Application Administrator som skal ha `Feide Organization ID`? Ref: https://github.com/BIBSYSDEV/NVA-scenarios/blob/master/1227-customer_registry/366-application_administrator_adds_an_institution.feature

Jo, det er et godt poeng at det kanskje er litt overkill med to scenario. Men det er jo to forskjellige ting en bruker gjør da. Hva tror dere, @Eiriknil og @terjehell ?

Reply:
jeg tror det blir enklere om vi begynner å legge flere scenario under samme feature, som vi har noen plasser nå også: https://github.com/BIBSYSDEV/NVA-scenarios/blob/master/26-register_publication/1694-creator_selects_reference_type_student_thesis_and_selects_subtype.feature

men @Eiriknil bør muligens ha siste ord her

Reply:
Enig at det blir enklere med flere scenarioer under samme feature der det er naturlig.